### DISCUSSION TITLE
How can i set custome header

### DISCUSSION THREAD

**Discussion by BKSoftCraft** [MicrosoftEdge/WebView2Feedback#4066] [Q&A]
I want to set custome header so that i can send auth token to server.
I tried,
````C#
webView.CoreWebView2.WebResourceRequested += CoreWebView2_WebResourceRequested;

private void CoreWebView2_WebResourceRequested(object? sender, CoreWebView2WebResourceRequestedEventArgs e)
{
    e.Request.Headers.SetHeader("Custome-Header", "This is my header");
}
````
but it is not working

**Comment by zhangliang2008** [comment]
[Intercepting a request, to monitor or modify it](https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/webresourcerequested?tabs=dotnet#example-intercepting-a-request-to-monitor-or-modify-it)
